README.md

```diff
@@ -166,6 +166,30 @@ $ bolt command run hostname --nodes 'sshnode*.example.com'
 
 Note that &mdash; in place of wildcard matching &mdash; shell-specific expansions (such as `sshnode{1,2}.example.com`) can also be useful ways to specify a pattern of nodes to run.
 
+### Querying PuppetDB for targets
+
+In addition to the inventory file, you can query PuppetDB to generate lists of targets.
+
+First, add the necessary configuration to `~/.puppetlabs/bolt.yaml`.
+
+```yaml
+puppetdb:
+  server_urls: ["https://puppetdb.example.com:8081"]
```

Comment:
This doesn't seem to work for me.

Reply:
Also, I know it's consistent with puppetdb.conf, but it's inconsistent with other Bolt config. *sigh*

Reply:
Might need to pull the `puppetdb` section out of config before passing it to the PuppetDB::Config object.

Reply:
Pushed a fix to actually pass the puppetdb section

Reply:
Is that something we can test very well? I guess this is best tested against a running PuppetDB.